src/sage/graphs/traversals.pyx

```diff
@@ -258,6 +267,9 @@ def lex_BFS(G, reverse=False, tree=False, initial_vertex=None, algorithm="fast")
         algorithm is in `O(n + m)`, and our implementation follows that
         complexity. See [HMPV2000]_ and next section for more details.
 
+    Loops and multiple edges are ignored during the computation of lex_BFS and
```

Comment:
either write `lex BFS` or ``` ``lex_BFS`` ```. Same for all lex_* methods.

Reply:
done